source/_integrations/influxdb.markdown

```diff
@@ -197,79 +249,193 @@ sensor:
 ```
 
 {% configuration %}
+api_version:
+  type: string
+  description: API version to use. Enter `1` or don't specify this field for 1.xx installations.
+  default: "1"
+ssl:
+  type: boolean
+  description: Use HTTPS instead of HTTP to connect.
+  required: false
+  default: false
 host:
-  description: IP address of your database host, e.g.,  192.168.1.10.
+  type: string
+  description: IP address of your database host, e.g., 192.168.1.10.
   required: false
   default: localhost
-  type: string
 port:
+  type: string
   description: Port to use.
   required: false
   default: 8086
+path:
   type: string
+  description: Path to use if your InfuxDB is running behind an reverse proxy.
+  required: false
 username:
+  type: string
   description: The username of the database user.
   required: false
-  type: string
 password:
-  description: The password for the database user account.
-  required: false
   type: string
-ssl:
-  description: Use HTTPS instead of HTTP to connect.
+  description: The password for the database user account.
   required: false
-  default: false
-  type: boolean
 verify_ssl:
+  type: boolean
   description: Verify SSL certificate for HTTP request.
   required: false
   default: false
-  type: boolean
 queries:
+  type: list
   description: List of queries.
   required: true
-  type: list
   keys:
     name:
+      type: string
       description: The name of the sensor.
       required: true
-      type: string
     unit_of_measurement:
+      type: string
       description: Defines the units of measurement of the sensor, if any.
       required: false
-      type: string
     measurement:
+      type: string
       description: Defines the measurement name in InfluxDB (the FROM clause of the query).
       required: true
-      type: string
     where:
-      description: Defines the data selection clause (the where clause of the query).
+      type: template
+      description: Defines the data selection clause (the where clause of the query). This supports [templates](/docs/configuration/templating/#building-templates).
       required: true
-      type: string
     value_template:
+      type: template
       description: Defines a [template](/docs/configuration/templating/#processing-incoming-data) to extract a value from the payload.
       required: false
-      type: template
     database:
+      type: string
       description: Name of the database to use.
       required: false
       default: home_assistant
-      type: string
     group_function:
+      type: string
       description: The group function to be used.
       required: false
       default: mean
-      type: string
     field:
+      type: string
       description: The field name to select.
       required: true
-      type: string
       default: value
 {% endconfiguration %}
 
+### Configuration for 2.xx Installations
```

Comment:
Why is this repeated with all options again?

I'm confused... a newer user would be as well I guess.

Reply:
My thinking was that it was different enough for 2.xx with the entirely different query language that it would be easier to just separate 1.xx and 2.xx configurations and examples. But I can merge and just put in `2.xx only` for things like I did other places.

Reply:
Ok merged the config sections, let me know what you think